.changeset/proud-coats-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+'legend-transactional': patch
+---
+
+update saga title
```

Comment:
Improve the description of the change/refactor/etc.. same or better that the **doc** section in the PR template. This is going to be part of CHANGELOG